bot/resources/tags/mutability.md

```diff
@@ -0,0 +1,32 @@
+**Mutable vs immutable objects**
+
+Imagine that you want to make all letters in a string upper case. Conveniently, strings have an `.upper()` method.
+
+You might think that this would work:
+```python
+string = "abcd"
+string.upper()
+print(string) # abcd
+```
+
+`string` didn't change. Why is that so?
+
+That's because strings in Python are _immutable_. You can't change them, you can only pass around existing strings or create new oness.
```

Comment:
```suggestion
That's because strings in Python are _immutable_. You can't change them, you can only pass around existing strings or create new ones.
```